### PR TITLE
Fixes to Stacks Blockchain Overview

### PR DESCRIPTION
**Pull request by reedrosenbluth**
Changed around some wording/phrasing. Tried to make it more comprehensible and consistent with our other messaging.

### CODE REVIEW

File: src/pages/stacks-blockchain/overview.md [hunk 1]
```diff
@@ -5,20 +5,20 @@ description: Learn more about the Stacks blockchain
 
 ## Introduction
 
-The Stacks 2.0 blockchain connects to Bitcoin for security and enables decentralized apps and predictable smart contracts. Stacks 2.0 implements a new mining mechanism called Proof of Transfer (PoX). PoX anchors to Bitcoin security and makes it possible to reward holders of Stacks (STX) tokens, the native token of the Stacks 2.0 blockchain, with bitcoins.
+Stacks 2.0 is a layer 1 blockchain that enables people to register digital assets and write smart contracts. It connects to Bitcoin with a 1:1 block ratio, meaning anything that happens on the Stacks blockchain can be verified on the Bitcoin Blockchain. It implements a new mining mechanism called Proof of Transfer (PoX). PoX uses the an established blockchain (in this case Bitcoin) to secure a new blockchain (Stacks).
```
Comment (agraebe) [inline src/pages/stacks-blockchain/overview.md:8]: I wonder if `layer 1 blockchain` will be well understood for folks without specific domain knowledge?

Comment (agraebe) [reply src/pages/stacks-blockchain/overview.md:8]: `PoX uses the an established` > `PoX uses an established`

File: src/pages/stacks-blockchain/overview.md [hunk 2]
```diff
@@ -31,17 +31,17 @@ You can view accounts, blocks, transactions, and smart contracts broadcasted to
...
 
-- **Predictability**: The Clarity language uses precise and unambiguous syntax that allows developers to predict exactly how their contracts will be executed
-- **Security**: The Clarity language allows users to supply their own conditions for transactions that ensure that a contract may never unexpectedly transfer a token owned by a user
-- **No compiling**: Contracts written in Clarity are broadcasted on the blockchain exactly as they are written by developers. This ensures that the code developers wrote, analyzed, and tested, is exactly what gets executed
+- **Predictability**: The Clarity language uses precise and unambiguous syntax that allows developers to predict exactly how their contracts will be executed.
+- **Security**: The Clarity language allows users to supply their own conditions for transactions that ensure that a contract may never unexpectedly transfer a token owned by a user.
+- **No compiler**: Contracts written in Clarity are broadcasted on the blockchain exactly as they are written by developers. This ensures that the code developers wrote, analyzed, and tested, is exactly what gets executed.
```
Comment (agraebe) [inline src/pages/stacks-blockchain/overview.md:44]: maybe worth aligning on the convention: I never end list items with `.`

Don't really have a preference but I'm certain we have inconsistencies in the docs

File: src/pages/stacks-blockchain/overview.md [hunk 3]
```diff
@@ -50,28 +50,28 @@ Smart contracts on the Stacks 2.0 blockchain are written in a new language calle
...
 [@page-reference | inline]
 | /mining
 
 ### Proof of Transfer (PoX)
 
-Proof of Transfer (PoX) is the consensus mechanism that leverages the security and stability of Bitcoin to create new blockchains, without modifying the Bitcoin protocol.
+Proof of Transfer (PoX) is a novel consensus mechanism that leverages the security and stability of Bitcoin to create a new blockchain, without modifying the Bitcoin protocol. 
 
-PoX makes it possible to reward holders of Stacks Token (STX) with Bitcoin: instead of spending energy, Stacks miners transfer Bitcoin to holders of Stacks Token (STX) to mine blocks. An implementation of this is called **Stacking**.
+STX miners transfer Bitcoin to other STX holders in order to mine blocks. This BTC reward mechanism is called **Stacking**.
```
Comment (agraebe) [inline src/pages/stacks-blockchain/overview.md:62]: We agreed a few weeks ago to always use `Stacks (STX)` when we reference the token. I'm seeing this inconsistency in different places in your PR. Could you adapt to the convention, please? :)

File: src/pages/stacks-blockchain/overview.md [hunk 4]
```diff
@@ -50,28 +50,28 @@ Smart contracts on the Stacks 2.0 blockchain are written in a new language calle
...
+STX token holders can help secure the network by participating in Stacking and locking up their STX tokens for a certain period of time. As a reward for this commitment, Stackers receive Bitcoin that miners transfer as part of Proof of Transfer (PoX).
 
 -> Proof of Transfer and Stacking are in active development and are coming soon
 
 ## Protocol specification
 
-The Stacks 2.0 blockchain specification consists a set of proposals, called Stacks improvement proposals (SIPs). Each SIP is aimed at describing the implementation of a part of the Stacks 2.0 blockchain. Together, all proposals contain concise technical specifications of features or standards and the rationale
-behind it.
+The Stacks 2.0 blockchain specification consists a set of proposals, called Stacks improvement proposals (SIPs). Each SIP is describes the implementation of a part of the Stacks 2.0 blockchain. Together, all proposals contain concise technical specifications of features or standards and the rationale
```
Comment (agraebe) [inline src/pages/stacks-blockchain/overview.md:72]: `Each SIP is describes the implementation` > `Each SIP describes the implementation`

**Review by agraebe — CHANGES_REQUESTED** [review]
thanks so much @reedrosenbluth! I left some minor requests in there

File: src/pages/stacks-blockchain/overview.md (hunk 2 above)
Comment (reedrosenbluth) [reply src/pages/stacks-blockchain/overview.md:44]: Gotcha. I tend to prefer adding the period when the line items are full sentences, and leaving it out when they are not

File: src/pages/stacks-blockchain/overview.md (hunk 1 above)
Comment (reedrosenbluth) [reply src/pages/stacks-blockchain/overview.md]: I feel good about layer 1 being accessible to most people who are reading through our docs, but will change if you feel strongly about it. I think it's important for differentiating us from the myriad of projects built on Ethereum.

Comment (agraebe) [reply src/pages/stacks-blockchain/overview.md]: all good, let's go for it :)

**Review by agraebe — APPROVED** [review]

**Pull request merged**